.github/workflows/ci.yml

```diff
@@ -15,7 +15,14 @@ jobs:
         with:
           node-version: 14
       - run: npm ci --no-optional --legacy-peer-deps
-      - run: npm test
+      - run: npm test -- --reporters=default --reporters=jest-junit
+      - name: Test Report
+        uses: dorny/test-reporter@v1
+        if: success() || failure() # run this step even if previous step failed
```

Comment:
can this line just be omitted to run it all the time? (I have never done more than tweak existing workflows so this may be a stupid question 😄 )